testsuite/mass-and-rinertia_per_particle.py

```diff
@@ -14,256 +14,176 @@
+        self.gamma_global = (0.5 + np.random.random(3)) * 2.0 / 3.0
+        self.gamma_global_rot = (0.2 + np.random.random(3)) * 20
+        # Per-particle parameters
+        self.kT_p = 2.5,2.0
         for k in range(2):
-            if test_case == 0:
-                if (k == 0):
-                    print("------------------------------------------------")
-                    print("Test " + str(test_case) + ": no particle specific values")
-                    print("------------------------------------------------")
-                # No assignments are needed.
-    
-            if test_case == 1:
-                if (k == 0):
-                    print("------------------------------------------------")
-                    print("Test " + str(test_case) +
-                        ": particle specific gamma but not temperature")
-                    print("------------------------------------------------")
-                self.es.part[k].gamma = gamma_tran[k, :]
-                if "ROTATION" in espressomd.features():
-                    self.es.part[k].gamma_rot = gamma_rot[k, :]
-    
-            if test_case == 2:
-                if (k == 0):
-                    print("------------------------------------------------")
-                    print("Test " + str(test_case) +
-                        ": particle specific temperature but not gamma")
-                    print("------------------------------------------------")
-                self.es.part[k].temp = 0.0
-    
-            if test_case == 3:
-                if (k == 0):
-                    print("------------------------------------------------")
-                    print("Test " + str(test_case) +
-                        ": both particle specific gamma and temperature")
-                    print("------------------------------------------------")
-                self.es.part[k].temp = 0.0
-                self.es.part[k].gamma = gamma_tran[k, :]
-                if "ROTATION" in espressomd.features():
-                    self.es.part[k].gamma_rot = gamma_rot[k, :]
-            
-            if test_case == 4:
-                if (k == 0):
-                    print("------------------------------------------------")
-                    print("Test " + str(test_case) + ": no particle specific values.")
-                    print("Rotational specific global thermostat")
-                    print("------------------------------------------------")
-                    # No assignments are needed.
-
-        if test_case == 1 or test_case == 3:
-            gamma_tr = gamma_tran
-            gamma_rot_validate = gamma_rot
-        else:
+            self.gamma_tran_p[k, :] = (0.4 + np.random.random(3)) * 10
+            self.gamma_rot_p[k, :] = (0.2 + np.random.random(3)) * 20
+        
+        ## Particles
+        self.mass = (0.2 + np.random.random()) * 7.0
+        self.J = (0.2 + np.random.random(3)) * 7.0
```

Comment:
where do the magic ```0.2``` and ```0.7``` come from? In general magic number should have a descriptive name